napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -445,29 +452,12 @@ def _on_color_mode_change(self):
                 self.colorModeComboBox.findData(self.layer.color_mode)
             )
 
-    def _on_editable_change(self):
-        """Receive layer model editable change event & enable/disable buttons."""
-        # In 3D mode, we need to disable all buttons other than picking
-        # (only picking works in 3D)
-        widget_list = [
-            'pick_button',
-            'fill_button',
-            'paint_button',
-            'erase_button',
-        ]
-        widgets_to_toggle = {
```

Comment:
I think this used to be relevant (see [the PR where it was introduced](https://github.com/napari/napari/pull/3074/files#diff-08dfc2d066c9a57e7a5fd908ad3ba987018ab8627944010291ec31cf12f36468R540)), but all the values of the dict are the same now. Right?